operators/test/e2e/apm/standalone_test.go

```diff
@@ -29,7 +27,6 @@ func TestApmStandalone(t *testing.T) {
 
 func TestApmStandaloneNoTLS(t *testing.T) {
 	apmBuilder := apmserver.NewBuilder("standalone-no-tls").
-		WithOutput(apmtype.Output{}).
```

Comment:
Would be cleaner if we would remove the `elasticsearchRef` for these tests imo otherwise it is not really a standalone test but a 'continuously waiting for ES test'

Reply:
Good catch ! 👍